src/evm/chandrupatla.rs

```diff
@@ -0,0 +1,1027 @@
+//! Chandrupatla's method for finding swap amounts to reach target prices.
```

Comment:
if we want this to be the default for protocolSim this needs to be a PR to Tycho-indexer repository

Reply:
And that's what we want :]